folium/features.py

```diff
@@ -49,10 +49,8 @@ class RegularPolygonMarker(Marker):
     radius: int, default 15
         Marker radius, in pixels
     popup: string or folium.Popup, default None
-        Input text or visualization for object. Can pass either text,
-        or a folium.Popup object.
-        If None, no popup will be displayed.
 
```

Comment:
Looks great - less verbose and still descriptive. Just need to remove this line (#52) between popup and popup description in docstring.